server.js

```diff
@@ -14,6 +15,8 @@ app.get("/", (req, res) => {
     res.send("Welcome to Instock server!");
```

Comment:
if it is for testing connection, remove from 14-16 or else line no 15

Reply:
Sure